classes/book.rb

```diff
@@ -0,0 +1,14 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  def add_rentals(date, book, _person)
+    @rentals << Rental.new(date, book, self)
+  end
```

Comment:
- [ ] Since you are in the Book class already, then the add_rental method should optimized like the following:
```suggestion
  def add_rentals(date, person)
    @rentals << Rental.new(date, self, person)
  end
```

Reply:
@descholar-ceo Thank you for the review and the alternate. I will update the code accordingly.